waspc/cli/Command/Clean.hs

```diff
@@ -0,0 +1,24 @@
+module Command.Clean
+    ( clean
+    ) where
+
+import System.Directory (doesDirectoryExist, removeDirectoryRecursive)
+import System.IO (hFlush, stdout)
+import Control.Monad.IO.Class (liftIO)
+
+import qualified StrongPath as SP
+import Command (Command)
+import Command.Common (findWaspProjectRootFromCwd)
+import qualified Common
+
+clean :: Command ()
```

Comment:
aha so this deletes .wasp directory. Would be cool if there was a comment saying what this does, `clean` is pretty general. We will anyhow have some explanation for this in cli, right?

Reply:
Yes we can put some explanation! My idea was that clean just cleans up whatever is out there. Ok, that ends up being .wasp directory, but user does not need to know that. They just say "ok stuff had gone wrong, look here is clean, I will run that". We could add some better explanation though, in the Usage. But we should improve Usage in general, I would leave that for later.